packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -41,12 +39,15 @@ interface UserHeaderItemProps {
 export function UserHeaderItem(props: UserHeaderItemProps): React.ReactElement {
     const { aboutModalLogo } = props;
 
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down("md"));
+
     const user = useCurrentUser();
     const [showAboutModal, setShowAboutModal] = React.useState(false);
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
     return (
-        <AppHeaderDropdown buttonChildren={user.name} startIcon={<Account />}>
+        <AppHeaderDropdown buttonChildren={isMobile ? undefined : user.name} startIcon={<Account />}>
```

Comment:
I believe the `startIcon` has some additional margin on one side, which would make the icon not centered, and look weird when there is no text. 

Is that the case? 
If so, maybe the icon should be moved into the `buttonChildren` on mobile.

Reply:
I adapted it. Is this how you intended the change?